model/src/CMakeLists.txt

```diff
@@ -126,6 +126,7 @@ set(SRCFILES constants.F90
              w3wdatmd.F90
              wav_comp_nuopc.F90
              wav_import_export.F90
+             wav_grdout.F90
```

Comment:
@mvertens I am getting the following build error:

```
CMake Error at CMakeLists.txt:145 (add_library):
  Cannot find source file:

    wav_grdout.F90
```

Test root:
/glade/scratch/altuntas/ERS.TL319_t061_wt061.GMOM_JRA_WD.cheyenne_intel.C.20220503_102633_h5asv0

Reply:
I think you'll want to remove the reference to wav_grdout here and in wav_comp_nuopc. That code belongs w/ the refactored w3iogoncmd.F90, which allows any of the output variables to be written to netcdf. 

I think CESM is not ready to bring those changes in yet, since it would require minor changes to your set_shel_inp to create the needed list of variable names.